aten/src/ATen/CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ file(GLOB native_cpp "native/*.cpp")
 file(GLOB native_mkl_cpp "native/mkl/*.cpp")
 file(GLOB native_mkldnn_cpp "native/mkldnn/*.cpp")
 file(GLOB vulkan_cpp "vulkan/*.cpp")
-file(GLOB native_vulkan_cpp "native/vulkan/api/*.cpp" "native/vulkan/*.cpp")
+file(GLOB native_vulkan_cpp "native/vulkan/api/*.cpp" "native/vulkan/ops/*.cpp" "native/vulkan/*.cpp")
```

Comment:
I need to fix build issues you encountered on the devserver as part of enabling this.